src/main/java/Java/Http/Methodexample/HttpMethod/Controller.java

```diff
@@ -1,5 +1,91 @@
 package Java.Http.Methodexample.HttpMethod;
 
+import io.micrometer.common.lang.NonNull;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+@org.springframework.stereotype.Controller
+@RestController
+
 public class Controller {
```

Comment:
PRの概要にも記載いただいていますが、ControllerとRestControllerは分けるのが良いですね。
また、クラス名をControllerとしてしまうのでなく、たとえばNameControllerのようにクラス名がかぶらないようにしたほうがいいかと思います。
```suggestion
@Controller
public class NameController {
```
```suggestion
@RestController
public class NameRestController {
```

Reply:
クラスを分けたところ、思ったように表示されました！
クラスの名前も変更しました。

Reply:
OKです！